src/manifest.js

```diff
@@ -279,11 +279,26 @@ export const addPropertiesToMaster = (master, uri) => {
       master.playlists[i].uri = phonyUri;
     }
   }
+  const audioOnlyMaster = isAudioOnly(master);
 
   forEachMediaGroup(master, (properties, mediaType, groupKey, labelKey) => {
     const groupId = `placeholder-uri-${mediaType}-${groupKey}-${labelKey}`;
 
+    // add a playlist array under properties
     if (!properties.playlists || !properties.playlists.length) {
+      // if we have an audio only master and this media group does not have a uri.
+      // check if it is located in the real list. If it is don't add placeholder
+      // properties as we should never switch to it.
```

Comment:
I think the not switching to it confused me a bit. Would you say this statement is accurate?

```suggestion
      // If the manifest is audio only and this media group does not have a uri, check
      // if the media group is located in the main list of playlists. If it is, don't add
      // placeholder properties as it shouldn't be considered an alternate audio track.
```